src/sentry/static/sentry/app/views/groupDetails/pluginExtras.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const PluginExtras = React.createClass({
```

Comment:
Off topic, educational question because I'm starting to learn React: feel free to ignore. Why did you define this as a `const`? Is that to prevent it from accidentally being overwritten? Is this a best practice?